src/store/item-store.js

```diff
@@ -55,4 +55,14 @@ store.registerHandler('STUDENT_FOUND', data => {
     }
     store.emitChange();
 });
+
+store.registerHandler('ITEM_DUEDATE_UPDATED', item => {
+    for(let i = 0; i < items.length; i++){
```

Comment:
I knew it was a thing, I have no reason for not using it...I'll go back and change this when I have a chance